src/index.html

```diff
@@ -10,5 +10,39 @@
   </head>
   <body>
     <h1>Moyo header</h1>
```

Comment:
Please, delete it:

![image](https://user-images.githubusercontent.com/86923089/189957150-a111e4f1-aa9a-4f64-ba67-cfbe3eee98fe.png)